### ISSUE TITLE
Can't disable embedded updates/dlcs

### ISSUE THREAD

**Issue by exp111** [Ryujinx/Ryujinx#3803]
#### Overview:
Make it possible to disable updates or dlcs that are embedded in the game files itself (multi-file/content nsps/xcis). Those are currently supported, but can't be specifically disabled.

Currently (Ruyjinx shows that a update is installed, but can't disable it):
![grafik](https://user-images.githubusercontent.com/17885980/198393055-d29466fb-db29-40ce-b11c-4189a02cc685.png)

#### Smaller Details: 
Unknown

#### Nature of Request:  
- Addition
  - yuzu already supports it

### Why would this feature be useful?
Allows users to toggle dlcs or updates if they want to downgrade while only keeping one file.

**Comment by EmulationFanatic** [comment]
The common sense solution is: don't merge the game and the update after you dump them.

**Comment by exp111** [comment]
This is a small use case but one that does exist nonetheless. So your 'solution' isnt helpful

**Comment by MetrosexualGarbodor** [comment]
You are asking for a solution for a problem that you have to go out of your way to create for yourself. This is why it's not a priority.

**Comment by jduncanator** [comment]
I feel like implementing support for this is step backward. The only reason merged content like this exists, is because of yuzu's lack of a proper update and DLC manager for so long, and the requirement to merge updates an DLC into games for them to work.

I believe yuzu now has a proper update and DLC management system, so it's probably wise to just extract your content into separate base game and updates.

**Comment by exp111** [comment]
Yeah this is mostly for the case that you use yuzu and ryujinx. Idk if they got a new update recently but afaik you still need to extract the updates to the fs which means you need to have double the space if you want to play them on both emus. 

But afaik there exist a few games that have embedded updates (even though you could split them), so it's not that outlandish to support it

**Comment by emmauss** [comment]
> Yeah this is mostly for the case that you use yuzu and ryujinx. Idk if they got a new update recently but afaik you still need to extract the updates to the fs which means you need to have double the space if you want to play them on both emus.
> 
> But afaik there exist a few games that have embedded updates (even though you could split them), so it's not that outlandish to support it

Those games comes with embedded updates for a reason, either as the update for dlc, or fixes and stuff. You aren't able to "uninstall" such updates on a switch, so why would we support removing those updates. The game should work  in whatever format it was released in.
